### ISSUE TITLE
`event.inspect` erroring out

### ISSUE THREAD

**Issue by railsjedi** [cronofy/cronofy-ruby#8]
> event.inspect
TypeError Exception: no implicit conversion of Time into String

**Comment by railsjedi** [comment]
tracing this issue now, i'll submit a pull request shortly

**Comment by railsjedi** [comment]
```
/Users/jc/.rvm/rubies/ruby-2.1.5/lib/ruby/2.1.0/time.rb:532:in `xmlschema': no implicit conversion of Time into String (TypeError)
	from /Users/jc/.rvm/gems/ruby-2.1.5/bundler/gems/cronofy-ruby-28e690c9b934/lib/cronofy/types.rb:34:in `coerce'
	from /Users/jc/.rvm/gems/ruby-2.1.5/gems/hashie-2.1.2/lib/hashie/extensions/coercion.rb:15:in `[]='
	from /Users/jc/.rvm/gems/ruby-2.1.5/gems/hashie-2.1.2/lib/hashie/extensions/coercion.rb:25:in `custom_writer'
	from /Users/jc/.rvm/gems/ruby-2.1.5/gems/hashie-2.1.2/lib/hashie/mash.rb:159:in `block in deep_update'
	from /Users/jc/.rvm/gems/ruby-2.1.5/gems/hashie-2.1.2/lib/hashie/mash.rb:152:in `each_pair'
	from /Users/jc/.rvm/gems/ruby-2.1.5/gems/hashie-2.1.2/lib/hashie/mash.rb:152:in `deep_update'
	from /Users/jc/.rvm/gems/ruby-2.1.5/gems/hashie-2.1.2/lib/hashie/mash.rb:67:in `initialize'
	from /Users/jc/.rvm/gems/ruby-2.1.5/gems/hashie-2.1.2/lib/hashie/mash.rb:132:in `new'
	from /Users/jc/.rvm/gems/ruby-2.1.5/gems/hashie-2.1.2/lib/hashie/mash.rb:132:in `dup'
	from /Users/jc/.rvm/gems/ruby-2.1.5/gems/hashie-2.1.2/lib/hashie/hash_extensions.rb:23:in `hashie_stringify_keys'
	from /Users/jc/.rvm/gems/ruby-2.1.5/gems/hashie-2.1.2/lib/hashie/hash_extensions.rb:40:in `hashie_inspect'
	from /Users/jc/Projects/ForwardTrail/growth/processing/processors/cronofy_processor.rb:12:in `block in import'
	from /Users/jc/.rvm/gems/ruby-2.1.5/bundler/gems/cronofy-ruby-28e690c9b934/lib/cronofy/client.rb:477:in `block in each'
	from /Users/jc/.rvm/gems/ruby-2.1.5/bundler/gems/cronofy-ruby-28e690c9b934/lib/cronofy/client.rb:476:in `each'
	from /Users/jc/.rvm/gems/ruby-2.1.5/bundler/gems/cronofy-ruby-28e690c9b934/lib/cronofy/client.rb:476:in `each'
	from /Users/jc/Projects/ForwardTrail/growth/processing/processors/cronofy_processor.rb:11:in `import'
	from /Users/jc/Projects/ForwardTrail/growth/app/workers/service_process_worker.rb:17:in `perform'
	from /Users/jc/Projects/ForwardTrail/growth/app/workers/concerns/worker.rb:48:in `perform'
	from /Users/jc/Projects/ForwardTrail/growth/bin/processors:57:in `<top (required)>'
	from /Users/jc/.rvm/gems/ruby-2.1.5/gems/railties-4.1.6/lib/rails/commands/runner.rb:60:in `load'
	from /Users/jc/.rvm/gems/ruby-2.1.5/gems/railties-4.1.6/lib/rails/commands/runner.rb:60:in `<top (required)>'
	from /Users/jc/.rvm/gems/ruby-2.1.5/gems/activesupport-4.1.6/lib/active_support/dependencies.rb:247:in `require'
	from /Users/jc/.rvm/gems/ruby-2.1.5/gems/activesupport-4.1.6/lib/active_support/dependencies.rb:247:in `block in require'
	from /Users/jc/.rvm/gems/ruby-2.1.5/gems/activesupport-4.1.6/lib/active_support/dependencies.rb:232:in `load_dependency'
	from /Users/jc/.rvm/gems/ruby-2.1.5/gems/activesupport-4.1.6/lib/active_support/dependencies.rb:247:in `require'
	from /Users/jc/.rvm/gems/ruby-2.1.5/gems/railties-4.1.6/lib/rails/commands/commands_tasks.rb:128:in `require_command!'
	from /Users/jc/.rvm/gems/ruby-2.1.5/gems/railties-4.1.6/lib/rails/commands/commands_tasks.rb:95:in `runner'
	from /Users/jc/.rvm/gems/ruby-2.1.5/gems/railties-4.1.6/lib/rails/commands/commands_tasks.rb:40:in `run_command!'
	from /Users/jc/.rvm/gems/ruby-2.1.5/gems/railties-4.1.6/lib/rails/commands.rb:17:in `<top (required)>'
	from /Users/jc/.rvm/gems/ruby-2.1.5/gems/activesupport-4.1.6/lib/active_support/dependencies.rb:247:in `require'
	from /Users/jc/.rvm/gems/ruby-2.1.5/gems/activesupport-4.1.6/lib/active_support/dependencies.rb:247:in `block in require'
	from /Users/jc/.rvm/gems/ruby-2.1.5/gems/activesupport-4.1.6/lib/active_support/dependencies.rb:232:in `load_dependency'
	from /Users/jc/.rvm/gems/ruby-2.1.5/gems/activesupport-4.1.6/lib/active_support/dependencies.rb:247:in `require'
	from /Users/jc/Projects/ForwardTrail/growth/bin/rails:11:in `<top (required)>'
	from /Users/jc/.rvm/gems/ruby-2.1.5/gems/activesupport-4.1.6/lib/active_support/dependencies.rb:241:in `load'
	from /Users/jc/.rvm/gems/ruby-2.1.5/gems/activesupport-4.1.6/lib/active_support/dependencies.rb:241:in `block in load'
	from /Users/jc/.rvm/gems/ruby-2.1.5/gems/activesupport-4.1.6/lib/active_support/dependencies.rb:232:in `load_dependency'
	from /Users/jc/.rvm/gems/ruby-2.1.5/gems/activesupport-4.1.6/lib/active_support/dependencies.rb:241:in `load'
	from /Users/jc/.rvm/rubies/ruby-2.1.5/lib/ruby/site_ruby/2.1.0/rubygems/core_ext/kernel_require.rb:54:in `require'
	from /Users/jc/.rvm/rubies/ruby-2.1.5/lib/ruby/site_ruby/2.1.0/rubygems/core_ext/kernel_require.rb:54:in `require'
	from -e:1:in `<main>'

```

**Comment by railsjedi** [comment]
i dug into it, couldnt figure it out. guess i can live without inspect for now :) 

`to_hash` works fine

**Comment by gshutler** [comment]
Thanks for the report, that was some strange behaviour. I wouldn't have expected a coerced value to be coerced again but the change required implies that's what was needed.

**Comment by gshutler** [comment]
Released in v0.5.3

**Comment by railsjedi** [comment]
:+1: awesome, thanks!
